### PR TITLE
refactor: fix hasOwnProperty and format code

### LINKED ISSUE #3735: Codacy: Improve Code Quality and Upgrade the grade to atleast A
Codacy is grading the Project Code Style as B , improve the code quality upto atleast A

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes #3735 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 7
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4747715&bid=15841179)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 7
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4747715&bid=15841179)

File: scripts/replace-config.js [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
+  const packageInfo = require('.' + packagePath);
 
-  const old = packageInfo.fastboot.config[appName]
+  const old = packageInfo.fastboot.config[appName];
   packageInfo.fastboot.config[appName] = merge(old, environment);
-  await promisify(fs.writeFile)(packagePath, JSON.stringify(packageInfo))
+  await promisify(fs.writeFile)(packagePath, JSON.stringify(packageInfo));
 
-  console.log('Transformed package.json with new environment')
+  //console.log('Transformed package.json with new environment')
```
Comment (iamareebjamal) [inline scripts/replace-config.js]: Revert

File: scripts/replace-config.js [hunk 2]
```diff
@@ -72,43 +72,43 @@ function findObject(js, objectStart) {
...
+    if (defaultIndex < 0) {
       continue;
-    
+    }
     // File with environment definition found
 
-    console.log('Transforming ' + js)
-    
-    const object = findObject(code, defaultIndex)
+    //console.log('Transforming ' + js)
```
Comment (iamareebjamal) [inline scripts/replace-config.js]: Revert

File: scripts/replace-config.js [hunk 3]
```diff
@@ -72,43 +72,43 @@ function findObject(js, objectStart) {
...
+
+    const object = findObject(code, defaultIndex);
 
-    console.log('Environment Object Found', object)
+    //console.log('Environment Object Found', object)
```
Comment (iamareebjamal) [inline scripts/replace-config.js]: Revert

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 7
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4747715&bid=15841179)

**Pull request merged**